Lib/test/test_sqlite3/test_regression.py

```diff
@@ -238,7 +231,8 @@ def test_auto_commit(self):
         2.5.3 introduced a regression so that these could no longer
         be created.
         """
-        con = sqlite.connect(":memory:", isolation_level=None)
+        with memory_database(isolation_level=None) as con:
```

Comment:
This does not feel complete: can we assert something here? 🤔

Reply:
That test has been like that for years. I'm not sure it is really helpful anymore.